### PR TITLE
Fetchcalls/ec

### PR DESCRIPTION
**Pull request by Emily-Cathleen**
# Description
Successfully added api calls. Tested along with the BE team, our GET and POST requests are working. 

## Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor
- [ ] Documentation update

## Check the correct box
- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Checklist:
- [x] I have linked the appropriate Project Board in this PR
- [x] My code follows the style guidelines of this project
- [x] My code has no unused/commented out code
- [x] I have performed a self-review of my code
- [x] I have fully tested my code

### CODE REVIEW

**Review by Jerry-Vrrr — APPROVED** [review]
We have data! Yay!

**Review by KATIETOLER — COMMENTED** [review]
Looks great, I'm really happy we got the fetch calls done.

File: App.js [hunk 1]
```diff
@@ -18,11 +18,10 @@ const App = () => {
 	const [modalVisible, setModalVisible] = useState(false)
 	const [cameraModalVisible, setCameraModalVisible] = useState(false)
 	const [allListings, setAllListings] = useState([])
-	
-	const createNewListing = (newListing) => {
-		console.log('newlistboi', newListing)
-		setAllListings([...allListings, newListing])
-	}
+
+	// const createNewListing = (newListing) => {
```
Comment (BrendaSotel0) [inline App.js:22]: Let's clean up all the comments when we get a chance.

**Review by BrendaSotel0 — COMMENTED** [review]
This is coming along great.

**Review by kammon10 — APPROVED** [review]

**Pull request merged**